docs/docs/getting-started/first-steps.md

```diff
@@ -184,7 +176,7 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
       Supported are all machines from the N2D family. It defaults to `n2d-standard-4` (4 vCPUs, 16 GB RAM), but you can use any other VMs from the same family. Refer to [N2D machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines) or run `constellation config instance-types` to get the list of all supported options.
 
     </tabItem>
-    <tabItem value="aws" label="AWS">
+    <tabItem value="aws" label="AWS (Console)">
```

Comment:
This doesn't use the AWS Console. We should name it differently, maybe "(manually)".